text/0025-purchase-order.md

```diff
@@ -566,8 +566,9 @@ message UpdatePurchaseOrderPayload {
 
 message CreateVersionPayload {
```

Comment:
In the same vein as this commit, I believe the `UpdatePurchaseOrderPayload` needs a `po_uid` field